ngraph/test/visitors/op/softplus.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "ngraph/opsets/opset1.hpp"
+#include "util/visitor.hpp"
+
+using namespace std;
+using namespace ngraph;
+using ngraph::test::NodeBuilder;
+using ngraph::test::ValueMap;
+
+TEST(attributes, softplus)
```

Comment:
Hi could you please rebase your PR to the latest master. And please use parametrized visitor API test which was introduced in this PR: https://github.com/openvinotoolkit/openvino/pull/6181

Reply:
done.